lib/travis/build/script/matlab.rb

```diff
@@ -0,0 +1,39 @@
+module Travis
+    module Build
+      class Script
+        class Matlab < Script
+          MATLAB_INSTALLER_LOCATION = 'https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh'.freeze
+          MATLAB_EXECUTABLE = 'matlab -batch'.freeze
+          MATLAB_COMMAND = "results = runtests('IncludeSubfolders',true," \
+                              "'IncludeReferencedProjects',true), " \
+                              'assert(all(~[results.Failed]))'.freeze
```

Comment:
DO we think that the installer will install 20a by the time this gets merged? If so, we should change this line to:

`"assertSuccess(runtests('IncludeSubfolders',true,'IncludeReferencedProjects',true)"`

Reply:
Going with your suggestion under @mcafaro's comment :)